tracer/src/Datadog.Trace/Ci/CiEnvironment/TravisEnvironmentValues.cs

```diff
@@ -27,11 +27,7 @@ protected override void OnInitialize(IGitInfo gitInfo)
         Tag = ValueProvider.GetValue(Constants.TravisTag);
         if (string.IsNullOrEmpty(Tag))
         {
-            Branch = ValueProvider.GetValue(Constants.TravisPullRequestBranch);
-            if (string.IsNullOrWhiteSpace(Branch))
-            {
-                Branch = ValueProvider.GetValue(Constants.TravisBranch);
-            }
+            Branch = ValueProvider.GetValue(Constants.TravisBranch);
```

Comment:
I guess this one was generally wrong too?

Reply:
Meh this failed a couple of shared tests we have across all libraries, so I'm rolling back this, until further discussion with the team. I don't want to change the current behaviour.